### DISCUSSION TITLE
Issue with IOS Build

### DISCUSSION THREAD

**Discussion by arungowda09** [axmolengine/axmol#2267] [Q&A] [answered]
I have added pods for my project Firebase , Admob , facebooksdk getting this issue  to build and I got stuck on this:

ld: framework 'Pods_Ludo' not found
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Ludo is my project name and Pods_Ludo is already listed there 'Target Support Files' in project navigator.

And Pods_Ludo.framework is also there in 'Frameworks, Libraries, and Embedded Content'

**Comment by rh101** [comment]
I've seen this problem before, but I can't quite recall what causes it.

Try adding this to your `CMakeLists.txt` to see if it helps:

```
set_target_properties(${APP_NAME} PROPERTIES
    XCODE_ATTRIBUTE_CONFIGURATION_BUILD_DIR "$(inherited)"
    XCODE_ATTRIBUTE_ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES "$(inherited)"
)
```

Please show the contents of the Podfile too.

**Reply by arungowda09** [reply, marked as answer]
Thank you for your reply. The issue was resolved by referring to this: https://github.com/axmolengine/axmol/discussions/1817.
